linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_files_permissions/bash/shared.sh

```diff
@@ -12,7 +12,7 @@ readarray -t RSYSLOG_INCLUDE < <(awk '/)/{f=0} /include\(/{f=1} f{nf=gensub("^(i
 declare -a LOG_FILE_PATHS
 
 declare -a RSYSLOG_CONFIGS
-RSYSLOG_CONFIGS+=("${RSYSLOG_ETC_CONFIG}" "${RSYSLOG_INCLUDE_CONFIG[@]}" "${RSYSLOG_INCLUDE[@]}")
+RSYSLOG_CONFIGS+=(${RSYSLOG_ETC_CONFIG} ${RSYSLOG_INCLUDE_CONFIG[@]} ${RSYSLOG_INCLUDE[@]})
```

Comment:
Shellcheck complains that RSYSLOG_ETC_CONFIG is not quoted, as it is not  an array I can probably stay quoted.

Reply:
But I think expansion of array with quotes should work. Maybe the problem is with expanding multiple arrays at once?
@matejak Can you provide more context on this change?

https://github.com/ComplianceAsCode/content/commit/33fa9020e7ee006df6c01812188c5e4335cd6c66#diff-b3a261983f580c0a14bfe622685710b716ddddb53814265f52bac6b39ffa0d67R15